api/controller/pokemon_controller.go

```diff
@@ -0,0 +1,36 @@
+package controller
+
+import (
+	"go-bootcamp/service"
```

Comment:
As a best practice, you can sort the imports in this categories:
```go
import (
   // Go core
   
   // Project
   
   // Thirdparty
)
```

Reply:
imports already sorted by category